src/SortBy/SortBy.jsx

```diff
@@ -1,16 +1,31 @@
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
 
 import styles from './SortBy.scss';
+import { setSortBy } from '../actions';
 
-const SortBy = ({ movieDetails }) => {
+const SortBy = ({ movieDetails, sortBy, dispatch }) => {
   const cx = classNames.bind(styles);
+  const handleSortBy = (value) => {
```

Comment:
Let's move `handleSortBy` outside of the `SortBy` function. The same is for `const cx = classNames.bind(styles);`. We don't need it to be created every time when component is rerendered